src/components/StructuredList/StructuredList.jsx

```diff
@@ -92,15 +96,15 @@ const StructuredList = ({ columns, data, design, isFixedWidth, onRowClick, loadi
           </StructuredListRow>
         ))}
       </StructuredListBody>
+      {!data.length ? (
+        <EmptyContent>
+          <Bee32 width={100} height={100} fill="#5a6872" />
```

Comment:
The `fill` hex color should come from the `COLORS` object in `styles.js`.

Reply:
Fixed on new commit